### DISCUSSION TITLE
PocketBase for E-commerce?

### DISCUSSION THREAD

**Discussion by naousislam** [pocketbase/pocketbase#1646] [Q&A] [answered]
Should I use PocketBase for an online store? Such as a clothing brand, where PocketBase will contain user and product information. Why or why not? Would there be scalability or security concerns?

Thanks,
Nawa

**Comment by silenceofthewans** [comment, marked as answer]
If this online store is mostly a database of products and prices, uses something like shopify and you are fine with sqlite and it's [limitations](https://www.sqlite.org/limits.html) than why not. Pocketbase will most likely be as scalable as the technologies it uses: (GO, sqlite) but according to [this](https://github.com/pocketbase/pocketbase/discussions/71#discussioncomment-3119545) discussion scalability isn't it's core focus. You can get more of an idea on pocketbase's current goals and limitations from these discussions: #1290, #417, #515 (_Also, pocketbase is in development stage and not 1.0 yet_)

For general security checklist, you can follow [OWASP cheatsheets](https://cheatsheetseries.owasp.org/IndexTopTen.html).

**Reply by naousislam** [reply]
Thank you for the reply, I've decided to go with MongoDB.
